apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/constants.ts

```diff
@@ -1,10 +1,18 @@
 export const FLOW_ID = 'gutenboarding';
-
+export const GUTENBOARDING_LAUNCH_FLOW = 'gutenboarding-launch';
+export const FOCUSED_LAUNCH_FLOW = 'focused-launch';
+export const SITE_LAUNCH_FLOW = 'launch-site';
 declare global {
 	interface Window {
 		wpcomEditorSiteLaunch?: {
+			launchUrl: string;
+			launchFlow: string;
+			isGutenboarding: boolean;
```

Comment:
When logging `siteLaunchOptions` for a site created using `/start` I see that `isGutenboarding` is actually an empty string.
<img width="320" alt="Screenshot 2021-01-27 at 09 04 07" src="https://user-images.githubusercontent.com/14192054/105955283-b6907b80-607e-11eb-8797-08bbdd189338.png">

Reply:
Looks like there's no way to use boolean (true/false) or number (1/0) via `wp_localize_script`, so we'll just have to agree that `""` is false and `"1"` is true.

In the consideration of whether to `"0"` or `""`, `""` would be preferable over `"0"` because `!""===true`  vs `!"0"===false`, so anyone `!isGutenboarding` would bring you right value.

I'll just update the type definition.

Reply:
Okay, scratch that. Changed my mind.
Now the isGutenboarding property either exists and hold the value "1" or does not exist at all.
The type definition is isGutenboarding?: '1';.
See eeca2eb.